stdlib/public/stubs/Stubs.cpp

```diff
@@ -232,8 +240,19 @@ extern "C" uint64_t swift_float80ToString(char *Buffer, size_t BufferLength,
 /// if an error occurred, or EOF was reached.
 SWIFT_RUNTIME_STDLIB_INTERFACE
 extern "C" ssize_t swift_stdlib_readLine_stdin(char **LinePtr) {
+#if defined(_MSC_VER)
+  if (std::cin.eof()) {
+    *LinePtr = nullptr;
+    return -1;
+  }
+  std::string Line;
+  std::getline(std::cin, Line);
+  *LinePtr = static_cast<char *>(std::malloc(Line.size() + 1));
+  return Line.size();
+#else
   size_t Capacity = 0;
   return getline(LinePtr, &Capacity, stdin);
```

Comment:
I realize it's not this patch's fault, but passing LinePtr (which is an invalidated out param) directly into getline is not defensive coding.   Either use a temp var or assert LinePtr is null.